include/sot/core/madgwickahrs.hh

```diff
@@ -111,8 +114,7 @@ namespace dynamicgraph {
 
       /* --- METHODS --- */
       double invSqrt(double x);
-      void madgwickAHRSupdateIMU
-	(double gx, double gy, double gz, double ax, double ay, double az) ;
+      void madgwickAHRSupdateIMU(double gx, double gy, double gz, double ax, double ay, double az) ;
```

Comment:
Please do not go over 80 lines. 
Redo the breaking line like it was.

Reply:
Sorry about that. Will do